src/main/java/techcourse/myblog/domain/Article.java

```diff
@@ -1,4 +1,69 @@
 package techcourse.myblog.domain;
 
+import java.util.Objects;
+
 public class Article {
+
+    private Long id;
+    private String title;
+    private String coverUrl;
+    private String contents;
+
+    public Article(String title, String coverUrl, String contents) {
+        this.title = title;
+        this.coverUrl = coverUrl;
+        this.contents = contents;
+    }
+
+    public Long getId() {
+        return id;
+    }
+
+    public void setId(Long id) {
+        this.id = id;
+    }
+
+    public String getTitle() {
+        return title;
+    }
+
+    public void setTitle(String title) {
```

Comment:
아래의 코드로 개선해 보면 어떨까요? setter 메서드는 도메인의 핵심 개념이나 의도를 코드에서 사라지게 해요.

```suggestion
    public void changeTitle(String title) {
```

Reply:
핵심 개념이나 의도를 코드에서 사라지게 만든다...음 ...맞는 말씀입니다.
질문)
만약 다른 곳에서 setter 메서드를 사용하고 있는 상황에서, 의도를 위해 setter메소드와 역할은 같지만 이름만 다른 changeTitle 메서드를 만드는건 어떻게 생각하시나요?
ex) 
public void setTitle(String title){
   this.title=title;
}
public void changeTitle(String title){
  this.title=title;
}